docs/usage.md

```diff
@@ -160,7 +178,30 @@ format.
 
 Example:
 ```
+./qsimh_amplitudes.x -c ../circuits/circuit_q40 -d 47 -k 0,1,2,3,4,5,6,7,8,9,10,13,14,15,16,17,23,24 -t 8 -w 0 -p 0 -r 4 -i ../circuits/bitstrings_q40_s1 -o ampl_q40_s1_w0 -v 1
```

Comment:
Please replace `-r 4` with `-r 13` and `ampl_q40_s1_w0` with `ampl_q40_s1`. It makes sense to describe how `-r` affects performance (perhaps in a later PR). There are 15 gates on the cut and there are two gates on the cut in the last cycle of two-qubit gates in this particular case. For better performance, it makes sense to perform the second checkpoint before these two gates.
In addition, could you perhaps replace `Usage` with `Usage of sample applications` or something like that in line 1?

Reply:
Done.